packages/auth/src/ResetPassword/ResetPasswordForm.tsx

```diff
@@ -47,14 +47,16 @@ export const ResetPasswordForm: FC<ResetPasswordFormProps> = (props) => {
           name='password'
           label={t('auth.forgot-password.new-password')}
           placeholder={t('auth.forgot-password.enter-your-password')}
+          data-test-id='newPass-box'
         />
         <FInput
           type='password'
           name='confirmPassword'
           label={t('auth.forgot-password.confirm-new-password')}
           placeholder={t('auth.forgot-password.enter-your-password-again')}
+          data-test-id='confirmPass-box'
```

Comment:
use the same data-test-id for as it appears in ProfilePasswordSettingPage it should be repeatPass-box or change the other file to be confirmPass-box